### DISCUSSION TITLE
Confusion when drawing SMILES with stereo-annotation

### DISCUSSION THREAD

**Discussion by bachi55** [rdkit/rdkit#4675] [Q&A] [answered]
Hi, 

I want to use RDKit (v 2021.03.4) to draw a set of molecules. Thereby I encountered that different tools and online resources lead to different depictions of the same molecule. Let's consider the following molecules: 

InChIKey: KWGRBVOPPLSCSI-WPRPVWTQSA-N
PubChem CID: [9294](https://pubchem.ncbi.nlm.nih.gov/compound/9294)
Isomeric SMILES (from PubChem): ```C[C@@H]([C@@H](C1=CC=CC=C1)O)NC```

I use the following code to draw the molecule (following [this](https://rdkit.blogspot.com/2020/04/new-drawing-options-in-202003-release.html) blog post):

```python
from rdkit.Chem import MolFromSmiles,
from rdkit.Chem.rdDepictor import Compute2DCoords
from rdkit.Chem.Draw import rdMolDraw2D
from IPython.display import SVG

mol = MolFromSmiles("C[C@@H]([C@@H](C1=CC=CC=C1)O)NC")
Compute2DCoords(mol)
d2d = rdMolDraw2D.MolDraw2DSVG(200, 200)
d2d.DrawMolecule(mol)
d2d.FinishDrawing()
SVG(d2d.GetDrawingText())
```

The output is:
![image](https://user-images.githubusercontent.com/9334594/140491460-f63d382f-9eef-418c-b2d4-127c5713c2d8.png)

However, if I compare this plot with different online-sources and the output of CDK it seems to be different:
**CDK**
![image](https://user-images.githubusercontent.com/9334594/140491662-51834520-2af3-4f57-b809-97878c54ff81.png)

[**ChemSpider**](https://www.chemspider.com/Chemical-Structure.8935.html)
![image](https://user-images.githubusercontent.com/9334594/140491792-9c1307a4-6c05-4fea-b202-817b28611628.png)

[**CompTex**](https://comptox.epa.gov/dashboard/dsstoxdb/results?search=DTXSID0022985)
![image](https://user-images.githubusercontent.com/9334594/140491883-14fcb212-a88a-46a1-91c6-55404c51a088.png)

Mainly I am confused by the bond stereo-annotation (wedged bonds). I am really not an chemist, but coming from the computer science with some knowledge in bioinformatics. So it might be just that I do not really understand how the bond stereo-annotation works, but shouldn't it be determined by the SMILES? How could the differences in the drawings be explained? 

Best regards,

Eric

**Comment by dehaenw** [comment, marked as answer]
These drawings all correspond to the same single stereoisomer, sometimes some of the bonds are rotated or the entire molecule is rotated, but the are the same molecule and the same stereoisomer, just a different 2D drawing. A good way to imagine this is that all regular bonds are in the plane (your screen, the paper, ..) including implicit Hs. All dashed bonds go below the plane (into the screen), all wedged bonds go above the plane (outside of the screen). If you would ask organic chemists to draw the structure of the example molecule you used "(−)-(1R,2S)-ephedrine" they would also not draw identical images necessarily.

**Reply by dehaenw** [reply]
here is the IUPAC document with a lot of information about the dashed/wedged drawing style: http://old.iupac.org/reports/provisional/abstract05/brecher_prs.pdf

**Reply by bachi55** [reply]
Thank for the clarification and the reference to the document!
